legacyflumeoffset/query.go

```diff
@@ -100,11 +100,11 @@ func (qry *lfoQuery) Reverse(yes bool) error {
 // 	if err != nil {
 // 		return errors.Wrap(err, "setCursorToLast: failed to establish current value")
 // 	}
-// 	currSeq, ok := v.(margaret.Seq)
```

Comment:
remove or comment why `setCursorToLast()` exists as a commented out block? maybe linking to an issue, if it's commented